### DISCUSSION TITLE
Replace icon font with individual SVGs

### DISCUSSION THREAD

**Discussion by brandonkelly** [craftcms/cms#8405] [Ideas]
Resources:

- https://twitter.com/SaraSoueidan/status/723163269695832064
- https://sarasoueidan.com/blog/icon-fonts-to-svg/
- https://blog.prototypr.io/align-svg-icons-to-text-and-say-goodbye-to-font-icons-d44b3d7b26b4#.84tpcpisz

(Would depend on a major CP rebuild like #1178.)

**Comment by missmatsuko** [comment]
This change would also improve accessibility since the SVG fonts are determined by CSS `content`, which may be announced to screen readers (and isn't translatable).

With SVG sprites, we wouldn't need to use the `svg()` function to avoid duplicate IDs (#7247) since the SVG instances are in the shadow DOM.

May want to use external SVG sprites, which are supported in our supported browsers:
https://fvsch.com/svg-symbol-support
